RecoTracker/IterativeTracking/python/LowPtQuadStep_cff.py

```diff
@@ -195,27 +198,27 @@
 
 
 # Final selection
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-lowPtQuadStep = TrackLwtnnClassifier.clone(
-    src = 'lowPtQuadStepTracks',
-    qualityCuts = [0.2, 0.425, 0.75],
-)
-fastSim.toModify(lowPtQuadStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
-
 from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+lowPtQuadStep = TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorLowPtQuadStep_Phase1'),
+     src = 'lowPtQuadStepTracks',
+     qualityCuts = [-0.7,-0.35,-0.15]
+)
 
-highBetaStar_2018.toReplaceWith(lowPtQuadStep, TrackMVAClassifierPrompt.clone(
+from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
+trackdnn.toReplaceWith(lowPtQuadStep, TrackLwtnnClassifier.clone(
     src = 'lowPtQuadStepTracks',
-    qualityCuts = [-0.9,-0.35,-0.15],
-    mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1') 
+    qualityCuts = [0.2, 0.425, 0.75]
 ))
 
-pp_on_AA_2018.toReplaceWith(lowPtQuadStep, TrackMVAClassifierPrompt.clone(
-    src = 'lowPtQuadStepTracks',
-    qualityCuts = [-0.9, -0.4, 0.3],
-    mva = dict(GBRForestLabel = 'HIMVASelectorLowPtQuadStep_Phase1')
-))
+highBetaStar_2018.toModify(lowPtQuadStep,qualityCuts = [-0.9,-0.35,-0.15])
```

Comment:
for future refs: this "simplification" also fixes a bug introduced in #21717 where DetachedTripletStep was passed incorrectly in this highBetaStar iter modifier.